modular_skyrat/modules/altborgs/code/robot_tipping.dm

```diff
@@ -0,0 +1,73 @@
+//	Modular solution for alternative skins and tipping
+/mob/living/silicon/robot/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/tippable/skyrat, \
+		tip_time = 3 SECONDS, \
+		untip_time = 2 SECONDS, \
+		self_right_time = 60 SECONDS, \
```

Comment:
```suggestion
		self_right_time = 15 SECONDS, \
```
As tempting as it is to lazily sit back on my bed and watch Skyrat alienate and burn off the last few borg players they've yet to get rid of, I still have a duty of care to the last two or three of them to quality control some of this stuff.
Letting some crybaby shitter give a borg a 60s stun if they stand still for a few seconds or start typing? No thanks, that can stay on TG.